.github/workflows/ci.yml

```
@@ -448,6 +454,7 @@ jobs:
     name: Quarkus Test Clustering
     needs: build
     runs-on: ubuntu-latest
+    timeout-minutes: 90
```

```suggestion
    timeout-minutes: 60
```

Last run was 22 min, so 60 should be more than sufficient.